packages/ui/src/components/ChannelListItem/index.tsx

```diff
@@ -77,11 +80,12 @@ function ChannelListItemIcon({
     );
   } else if (model.type === 'dm') {
     return (
-      <ListItem.AvatarIcon
-        backgroundColor={'red'}
-        contactId={model.members?.[0]?.contactId ?? model.id}
-        contact={model.members?.[0]?.contact}
-      />
+      <View borderRadius="$2xl" overflow="hidden">
+        <ListItem.AvatarIcon
```

Comment:
Avatar border radius is linked to its `size` prop -- would using that here instead of the wrapper work?

Reply:
It is, but I didn't see something fully-rounded in the Avatar component. I added a default-false "rounded" prop and passed it down.